LoM-Install/arista/do-install.py

```diff
@@ -0,0 +1,768 @@
+            logger.log("Error while checking if {} is running: {}".format(instance_name, e))
+            raise e
+
+        if not running:
+            logger.log("{} is not running".format(instance_name))
+            return False
+        logger.log("{} is running successfully".format(instance_name))            
+    return True
+    #To-Do : Goutham : call is_process_port_in_namespace() to check
+        
+def pre_checks():
+    """
+    This function is responsible for performing pre-installation checks.
+
+    It performs the following steps:
+    1. Check if each configuration file exists
+    2. Check if LoMEngine binary exists
+    3. Check if LoMPluginMgr binary exists
+    4. Check if LoMCli binary exists
+    5. Check if the TerminAttr daemon is running or not
+
+    Returns:
+    bool: True if all checks pass, False otherwise.
+    """
+    try:
+        # Check if each configuration file exists
+        config_files = [PROC_CONF_FILE, ACTIONS_CONF_FILE, BINDINGS_CONF_FILE, GLOBALS_CONF_FILE]
+        config_file_path = None
+        for config_file in config_files:
+            config_file_path = os.path.join(g_installer_dir, 'config', config_file)
+            if not os.path.exists(config_file_path):
+                logger.log("Error: Configuration file {} not found.".format(config_file))
+                return False
+
+        # Check if LoMEngine binary exists    
+        if not os.path.exists(g_lom_engine_path):
+            logger.log("Error: LoMEngine binary not found.")
+            return False
+
+        # Check if LoMPluginMgr binary exists
+        if not os.path.exists(g_lom_plugin_mgr_path):
+            logger.log("Error: LoMPluginMgr binary not found.")
+            return False
+
+        # Check if the LoMCli binary exists
+        if not os.path.exists(g_lom_cli_path):
+            logger.log("Error: LoMCli binary not found.")
+            return False
+
+         # Check if TerminAttr daemon is running or not
+        running = g_switch_eapi_handler.is_daemon_running("TerminAttr")
+        if not running:
+            # Just check if the TerminAttr config is present or not
+            exists = g_switch_eapi_handler.is_daemon_config_exists("TerminAttr")
+            if exists:
+                logger.log("TerminAttr daemon is not running, but daemon config is present in running config")            
+            logger.log("TerminAttr daemon is not running & also daemon config not present in running config")  
+            return False          
+        else:
+            logger.log("TerminAttr daemon is running")
+            # get the  TerminAttr daemon details
+            terminattrConfig = g_switch_eapi_handler.extract_terminattr_config()
+            logger.log("TerminAttr daemon details: {}".format(json.dumps(terminattrConfig)))
+
+        return True
+    except Exception as e:
+        logger.log("Error during pre-checks: {}".format(e))
+        raise e
+
+def post_checks():
```

Comment:
To-Do comment : Check for heartbeats, etc in future via eventd and via gnmi server.

Reply:
added